libkpod/layer/layer.go

```diff
@@ -0,0 +1,12 @@
+package layer
+
+import cstorage "github.com/containers/storage"
+
+// FullID gets the full id of a layer given a partial id or name
+func FullID(store cstorage.Store, id string) (string, error) {
```

Comment:
We wanted the full length iID for calls to `store.Changes()` or `store.Diff()`.  If those can accept names and partial IDs then we won't need this, otherwise we should leave it